README.md

```diff
@@ -151,20 +165,16 @@ just build-images
 
 # spin up the kurtosis devnet on your machine
 just up
-
-# run the web demo servers.
-just demo
 ```
 
 The web demo will be available on your browser at [`http://localhost:3000`](http://localhost:3000).
```

Comment:
Is it still available without `just demo`?

Reply:
sory, what is available?

Reply:
removed ref